foundry/game/File.py

```diff
@@ -98,3 +98,6 @@ def bulk_read(self, count: int, position: int) -> bytearray:
     def bulk_write(self, data: bytearray, position: int):
         position = self.prg_normalize(position)
         ROM.rom_data[position : position + len(data)] = data
+
+    def search(self, needle: bytes) -> int:
```

Comment:
I would add a start offset parameter, so that we can only search in a specific bank, since this only returns the first match.

Reply:
Resolved by f976918